pkg/splunk/enterprise/util_test.go

```diff
@@ -119,7 +119,7 @@ func TestGetLicenseMasterURL(t *testing.T) {
 	want = []corev1.EnvVar{
 		{
 			Name:  "SPLUNK_LICENSE_MASTER_URL",
-			Value: "splunk-stack1-license-master-service.test.svc.cluster.local",
+			Value: splcommon.Stack1LMClusterLocal,
```

Comment:
probably better to create/use `fmt.Sprintf()` format to create the string?
So lets say, 
```
in pkg/splunk/common ----
LMClusterLocal= "splunk-%s-license-master-service.test.svc.cluster.local"

and here in this pkg ----
Value: fmt.Sprintf(splcommon.LMClusterLocal, "stack1")
```